.env

```diff
@@ -67,3 +67,7 @@ HYKU_RESTRICT_CREATE_AND_DESTROY_PERMISSIONS=true
 # AND comment this out
 HYRAX_ANALYTICS=false
 ##END## Enable Google Analytics
+
+# When set to false, this turns off turbolinks which allows for
+# redirects from AF works to Valyrie works in the edit and show
+TURBOLINKS_FOR_VALKYRIE=false
```

Comment:
I wish this var had a more descriptive name, but I'm struggling to come up with a great alternative... 

I can imagine a dev seeing "TURBOLINKS_FOR_VALKYRIE", thinking "turbolinks makes things faster, I should set that to `true`", and unknowingly causing problems. 

Also, I think it should have a `HYKU_` prefix since that seems to be the convention.

Reply:
Maybe something along the lines of 

```
HYKU_ALLOW_REDIRECT_TO_VALKYRIE_RESOURCE=true
```

Reply:
I can make it with the HYKU prefix, but I'm also hoping they would read my comment

Reply:
Sounds good to me